src/battle/battle_icon.c

```diff
@@ -72,37 +71,37 @@ __attribute__((aligned(4))) static const u8 BattleIcon_PaletteIndexMap[] = {
 };
 
 /*
-* Maps move types (Physical, Special, Status) to the corresponding NARC member in pl_batt_obj.narc containing the type icon.
-*/
+ * Maps move types (Physical, Special, Status) to the corresponding NARC member in pl_batt_obj.narc containing the type icon.
+ */
 __attribute__((aligned(4))) static const u32 BattleIcon_MoveType_NARCMemberMap[] = {
     0xF4,
     0xF6,
     0xF5
 };
 
 /*
-* Maps a move type (Physical, Special, Status) to the corresponding palette index in NARC_INDEX_BATTLE__GRAPHIC__PL_BATT_OBJ__TYPE_PALETTE.
-*/
+ * Maps a move type (Physical, Special, Status) to the corresponding palette index in NARC_INDEX_BATTLE__GRAPHIC__PL_BATT_OBJ__TYPE_PALETTE.
+ */
 __attribute__((aligned(4))) static const u8 BattleIcon_MoveType_PaletteIndexMap[] = {
     0x0,
     0x1,
     0x0
 };
 
 /*
-* Given a type index, return the corresponding NARC member containing the icon for that type.
-* See also BattleIcon_GetNARCIndex.
-*/
+ * Given a type index, return the corresponding NARC member containing the icon for that type.
+ * See also BattleIcon_GetNARCIndex.
+ */
 u32 BattleIcon_TypeIndexToNARCMember(int typeIndex)
 {
     GF_ASSERT(typeIndex < NELEMS(BattleIcon_NARCMemberMap));
     return BattleIcon_NARCMemberMap[typeIndex];
 }
 
 /*
-* Returns the NARC member within pl_batt_obj containing palettes for battle icons.
-* See also BattleIcon_GetNARCIndex.
-*/
+ * Returns the NARC member within pl_batt_obj containing palettes for battle icons.
+ * See also BattleIcon_GetNARCIndex.
+ */
 u32 BattleIcon_GetPaletteNARCMember(void)
 {
     return NARC_INDEX_BATTLE__GRAPHIC__PL_BATT_OBJ__TYPE_PALETTE;
```

Comment:
**polish**: If you use the `naix` suggestion above, then you'll be able to get rid of this custom definition.